internal_transfers/actions/tests/database.spec.ts

```diff
@@ -40,4 +47,39 @@ describe("insertSettlementEvent(txHash, solver)", () => {
       },
     ]);
   });
+  test("insertImbalances(txHash, imbalances) & idempotent", async () => {
+    const txHash = "0x4321";
+    const token1 = "0x12";
+    const token2 = "0x34";
+    const imbalances = [
+      {
+        token: token1,
+        amount:
+          115792089237316195423570985008687907853269984665640564039457584007913129639935n,
+      },
+      { token: token2, amount: 5678n },
+    ];
+    await insertTokenImbalances(db, txHash, imbalances);
+
+    let results = await db.query(sql`SELECT * from internalized_imbalances;`);
+    const expectedResults = [
```

Comment:
Is it a given that the order is preserved?

Reply:
I suppose it isn't although I would think we can assume it is for now and if the tests start to fail we can fix it... Also, I don't believe that order matters at all for our purposes.

Reply:
In this case, I'd test it ignoring the order, so that the test reflects our expectations on what the function does. It might be enough to just change your test condition from `toStrictEqual` to `has.deep.member` ([docs](https://www.chaijs.com/api/bdd/#method_members), assuming Chai is used).

Reply:
Cool will do.